internal/osutils/osutils.go

```diff
@@ -56,10 +57,33 @@ func BashifyPath(absolutePath string) (string, error) {
 		return "", errs.New("Unrecognized absolute path format: %s", absolutePath)
 	}
 
-	absolutePath = strings.ToLower(absolutePath[0:1]) + absolutePath[2:]
-	absolutePath = strings.Replace(absolutePath, `\`, `/`, -1)  // backslash to forward slash
-	absolutePath = strings.Replace(absolutePath, ` `, `\ `, -1) // escape space
-	return "/" + absolutePath, nil
+	winPath, err := winPathToLinPath(absolutePath)
+	if err == nil {
+		winPath = strings.Replace(winPath, ` `, `\ `, -1) // escape space
+		return winPath, nil
+	}
+	logging.Error("Failed to bashify path using installed bash executable, falling back to slash replacement: %v", err)
+
+	vol := filepath.VolumeName(absolutePath)
+	absolutePath = absolutePath[len(vol):]
+	vol = strings.Replace(vol, ":", "", 1)
+	winPath = "/" + vol + filepath.ToSlash(absolutePath)
+	winPath = strings.Replace(winPath, ` `, `\ `, -1) // escape space
+	return winPath, nil
+}
+
+func winPathToLinPath(name string) (string, error) {
+	cmd := exec.Command("bash", "-c", "pwd")
+	cmd.Dir = filepath.Dir(name)
+
+	out, err := cmd.Output()
+	if err != nil {
+		return "", err
+	}
+
+	path := strings.TrimSpace(string(out)) + "/" + filepath.Base(name)
```

Comment:
Wouldn't `out` already contain `filepath.Base(name)` ?

Reply:
No, in line 77, we remove the basename, so we can change into that directory.